src/status_im/chat/models/mentions.cljs

```diff
@@ -167,17 +168,18 @@
         cursor                           (+ at-sign-idx (count primary-name) 2)]
     (rf/merge
      cofx
-     {:db                   (-> db
-                                (assoc-in [:chats/mention-suggestions chat-id] nil))
-      :set-text-input-value [chat-id new-text text-input-ref]
-      :dispatch             [:chat.ui/set-chat-input-text new-text chat-id]}
+     {:db       (-> db
+                    (assoc-in [:chats/mention-suggestions chat-id] nil))
+      :dispatch [:chat.ui/set-chat-input-text new-text chat-id]}
      ;; NOTE(rasom): Some keyboards do not react on selection property passed to
      ;; text input (specifically Samsung keyboard with predictive text set on).
      ;; In this case, if the user continues typing after the programmatic change,
      ;; the new text is added to the last known cursor position before
      ;; programmatic change. By calling `reset-text-input-cursor` we force the
      ;; keyboard's cursor position to be changed before the next input.
-     (reset-text-input-cursor text-input-ref cursor)
+     (when-not config/new-composer-enabled?
+       {:set-text-input-value [chat-id new-text text-input-ref]}
+       (reset-text-input-cursor text-input-ref cursor))
      (recheck-at-idxs public-key))))
```

Comment:
can you explain 180-182 ? it looks a little strange for me

Reply:
@qfrank this is some code that is used for the old composer. It is not needed for the new composer, but still has to keep it for now until the new composer is complete.

Reply:
i didn't see effect `:set-text-input-value` will be taken?

Reply:
`{:set-text-input-value [chat-id new-text text-input-ref]}` will be executed when `config/new-composer-enabled?` is false

Reply:
are you sure? have you tested [it](https://github.com/status-im/status-react/blob/b29b479b2c459e344fe4b1e4197046fcaed4e4d4/src/status_im/ui2/screens/chat/composer/input.cljs#L230) will be invoked? 🤔 @OmarBasem

Reply:
You are right @qfrank, fixed it. Thanks! This code will be removed in the next PR anyways :)